src/utilities/formInputConfigs.js

```diff
@@ -92,6 +117,22 @@ const FORM_INPUT_CONFIGS = seedObject => ({
     label: formInputStrings.date_of_birth,
     isEditable: true,
   },
+  [FORM_INPUT_KEYS.SEARCH_DATE_OF_BIRTH]: {
+    type: 'date',
+    initialValue: '',
+    key: 'dateOfBirth',
+    invalidMessage: formInputStrings.must_be_a_date,
+    isRequired: false,
+    validator: input => {
```

Comment:
Similar to existing date logic, but `validator` allows a blank input to enable the use case where a `dateOfBirth` is not specified.